runtime-modules/storage/src/weights.rs

```diff
@@ -37,6 +37,7 @@
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
+#![allow(unused_variables)]
```

Comment:
If you add this directive you can safely discard adding the underscore to `i`, `j`